docs/webactions.md

```diff
@@ -286,29 +284,99 @@ Notice in this case the JSON content is base64 encoded because it is treated as
 
 ### Enabling raw HTTP handling
 
-Raw HTTP web actions are enabled via the annotation `raw-http` [annotation](annotations.md) with a value of `true`.
+Raw HTTP web actions are enabled via the `--web` flag using a value of `raw`.
 
 ```bash
-$ wsk action create /guest/demo/hello hello.js \
-      --annotation web-export true
-      --annotation raw-http true
+$ wsk action create /guest/demo/hello hello.js --web raw
 ```
 
-**Note:** Since `raw-http` implies `web-export`, we plan to improve the CLI to provide a more convenient way to add (and remove) these annotations in the future.
+### Disabling raw HTTP handling
 
+Disabling raw HTTP can be accomplished by passing a value of `false` or `no` to the `--web`.
 
-### Disabling raw HTTP handling
+```bash
+$ wsk update create /guest/demo/hello hello.js --web false
+```
 
-Disabling raw HTTP is accomplished by setting the `raw-http` [annotation](annotations.md) value to `false`.
+### Decoding binary body content from Base64
+
+When using raw HTTP handling, the `__ow_body` content will be encoded in Base64 when the request content-type is binary.
+Below are examples demonstrating how to decode the body content in Node, Python, and Swift.
+
+#### Node
+
+```javascript
+$ cat decode.js
+function main(args) {
+    decoded = new Buffer(args.__ow_body, 'base64').toString('utf-8')
+    return {body: decoded}
+}
+```
 
 ```bash
-$ wsk update create /guest/demo/hello hello.js \
-      --annotation web-export true
-      --annotation raw-http false
+$ wsk action create nodeDecode decode.js --web raw
+ok: created action nodeDecode
+$ curl -k -H "content-type: application" -X POST -d "Decoded body" https://${APIHOST}/api/v1/web/guest/default/nodeDecode.json
+{
+  "body": "Decided body"
+}
 ```
 
-**Note:**  All annotations for a single action must be set at the same time, either when creating or updating the action. This is due to a current limitation on the API and CLI. Failure to do so will result is removal of any previously attached annotations.
 
+#### Python
+
+```python
+$ cat decode.py
+def main(dict):
+    try:
+        decoded = dict['__ow_body'].decode('base64').strip()
+        return {"body": decoded}
+    except:
+        return {"body": "Could not decode body from Base64."}
+```
+
+```bash
+$ wsk action create pythonDecode decode.py --web raw
+ok: created action pythonDecode
+$ curl -k -H "content-type: application" -X POST -d "Decoded body" https://${APIHOST}/api/v1/web/guest/default/pythonDecode.json
```

Comment:
since the response is going to be same in all cases, and since since the curl command is also the same... consider instead just showing the three functions and then just one example of how to curl it.